src/ol/layer/WebGLTile.js

```diff
@@ -255,9 +255,6 @@ function parseStyle(style, bandCount) {
  * property on the layer object; for example, setting `title: 'My Title'` in the
  * options means that `title` is observable, and has get/set accessors.
  *
- * **Important**: after removing a `WebGLTile` layer from your map, call `layer.dispose()`
- * to clean up underlying resources.
```

Comment:
Cleanup of the canvas and associated resources happens when you remove a layer from the map now.